src/in_memory_store.rs

```diff
@@ -92,46 +92,15 @@ impl<Address: Hash + Eq + Debug + Clone, Value: Clone + Eq + Debug> IntoIterator
     }
 }
 
+#[cfg(feature = "test-utils")]
 #[cfg(test)]
```

Comment:
Something like that is better:

```suggestion
#[any(cfg(test), cfg(feature = "test-utils")]
```